### PR TITLE
Finished the Affiliations page

### PR DESCRIPTION
**Pull request by MuzammilC**
## Description

This PR involves the development of the affiliations page for the website.

## Updates

[Please enter a brief summary of updates to this PR during code review - if any]

## Checklist

- [x] Ran `npm run lint:fix` to format code
- [x] Requested at least one reviewer
- [x]  Connected PR to Trello ticket (steps below)
- [ ] Addressed code review comments
- [ ] Gained at least one approval for your PR

## Connecting your PR to the corresponding ticket:
- Click on the "GitHub" button found on the right of your Trello ticket
- Choose "Attach PR" from the dropdown list
- Link your GitHub account (if you haven't done so already)
- Navigate to`VCL-content-platform` and choose your PR from the dropdown.

### CODE REVIEW

**Comment by MuzammilC** [comment]
[affiliated lab page (mobile view + tablet view + pc)](https://trello.com/c/MPyEoZNN/211-affiliated-lab-page-mobile-view-tablet-view-pc)

File: src/statics/affiliations.ts [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                },
+                
+            ]
+        },
+        {
+            type: "Other Universities",
+            content: [
+                {
+                    title: "Perception and Cognition Lab (Yale University)",
+                    href: "http://perception.yale.edu.s3-website-us-east-1.amazonaws.com/",
```
Comment (ElisaHu1) [inline src/statics/affiliations.ts:54]: change this to https://perception.yale.edu/

**Review by ElisaHu1 — APPROVED** [review]
looks great! just linting issue

**Review by ryanyae — CHANGES_REQUESTED** [review]
Sorry for the late review, but the only thing I would like a tweak of is adding max widths to the page, adding this will allow people with really large monitors to just look at the middle of their screen instead of having to map through their entire screen.

**Comment by ryanyae** [comment]
@MuzammilC Now that you have implemented the max-width you would have to account for this scenario
image

**Comment by MuzammilC** [comment]
The above issues should be fixed now, I also added a breakpoint for the width property of affiliations nav so that it looks cleaner on bigger screens.

**Review by ryanyae — APPROVED** [review]
Looks good to me, thank you Muzammil!

**Pull request merged**